embedded-graphics/src/primitives/circle.rs

```diff
@@ -79,6 +79,24 @@ impl Circle {
     pub fn center(&self) -> Point {
         self.top_left + Size::new(self.diameter, self.diameter) / 2
     }
+
+    /// Return the center point of the circle scaled by a factor of 2
+    ///
+    /// This method is used to accurately calculate the outside edge of the circle.
+    /// The result is not equivalent to `self.center() * 2` because of rounding.
+    fn center_2x(&self) -> Point {
+        self.top_left * 2 + self.size_minus_one()
+    }
+
+    //FIXME: This temporary method replicates the old broken conversion from diameter to size and
+    //should be removed.
```

Comment:
As with the other comment, please open an issue if this won't be fixed in the immediate future.